File: Backend/src/main/java/com/example/backend/HealthForm.java
package com.example.backend;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class HealthForm extends Order{

    private boolean answerStatus;
    private boolean oldStatus;
    private String[] symptomsList;

    public HealthForm(String place, String time, String date, User owner, boolean answerStatus, boolean oldStatus, String[] symptomsList){
        super(place, time, date, owner,"HealthForm");
        this.answerStatus = answerStatus;
        this.oldStatus = oldStatus;
        this.symptomsList = symptomsList;
    }
}

File: Frontend/src/classes/HealthForm.ts
import Order from './Order'
import User from './User'

export default class HealthForm extends Order {
  private _answerStatus: boolean;
  private _oldStatus: boolean;
  private _symptomsList: string[];

  public constructor(place: string, time: string, date: string, owner: string, answerStatus: boolean, oldStatus: boolean, symptomsList: string[], OID:string) {
    super(place, time, date, owner,"HealthForm");
      this._answerStatus = answerStatus;
      this._oldStatus = oldStatus;
      this._symptomsList = symptomsList;
  }

  public get answerStatus(): boolean {
    return this._answerStatus;
  }
  
  public set answerStatus(value: boolean) {
    this._answerStatus = value;
  }

  public get oldStatus(): boolean {
    return this._oldStatus;
  }
  
  public set oldStatus(value: boolean) {
    this._oldStatus = value;
  }

  public get symptomsList(): string[] {
    return this._symptomsList;
  }
  
  public set symptomsList(value: string[]) {
    this._symptomsList = value;
  }
}